### DISCUSSION TITLE
Mapbox API to provide coordinates (boundaries) to draw the polygon on react native

### DISCUSSION THREAD

**Discussion by codemap01** [rnmapbox/maps#1894] [Q&A]
We have a requirement for react native mobile app where we need to select (search) a country or city on the map we need to highlight that city and country and also need an API that provides (Coordinates for draw the polygon) I have explored and we are not able to identify that API to search and provides an array of Coordinates the city and country So Can you suggest any Mapbox API which fulfils our requirements because we need this type of API  and 
we have to find geocoding API for searching and we still need an API that provides Coordinates for draw the polygon  like this type on the image
I have attached a sample image.
<img width="1283" alt="Mapbox Sample API" src="https://user-images.githubusercontent.com/105269546/167620001-1cb6dce5-5212-473e-a500-1d4f18bb4653.png">

**Comment by VolkerLieber** [comment]
That seems to be possible with the boundary api.
Tutorial: [https://docs.mapbox.com/help/tutorials/point-in-polygon-query-with-mapbox-boundaries/](https://docs.mapbox.com/help/tutorials/point-in-polygon-query-with-mapbox-boundaries/)
Example: [https://demos.mapbox.com/boundaries-explorer/](https://demos.mapbox.com/boundaries-explorer/)

**Comment by rastapasta** [comment]
As an alternative: create an own back-end service with data from OSM, either exported from the [GeoFrabrik](https://download.geofabrik.de/) OSM exports or by using OverPass (or a similar OSM service/API):
![Screenshot 2022-05-15 at 21 05 34](https://user-images.githubusercontent.com/1259904/168489749-3fa74bdd-65fe-43cb-a521-93c0f4939f79.png)
